### PR TITLE
Refactor code generation

### PR DESCRIPTION
**Pull request by vmarkovtsev**
- Ability to generate from unpredicted vnodes
- Random code improvements

Signed-off-by: Vadim Markovtsev <vadim@sourced.tech>

### CODE REVIEW

**Comment by zurk** [comment]
@vmarkovtsev I cannot assign you to review, but it is ready.
The code works with the fix https://github.com/src-d/lookout-sdk-ml/pull/7

As soon as I solve https://github.com/src-d/style-analyzer/pull/335 I am going to add tests.

File: lookout/style/format/code_generator.py [hunk 1]
```diff
@@ -74,13 +74,7 @@ def apply_predicted_y(self, vnodes: Sequence[VirtualNode],
...
         for vnode, y_new in self._iterate_vnodes(vnodes, vnodes_y, y_pred):
             vnode = vnode.copy()
-            # if there is no prediction (y_new) for current VirtualNode (vnode)
-            # its value is set to None.
-            if y_new == vnode.y and not vnode.value:
-                # Skipping all unmodified NOOPS and empty tokens
-                # No need to finalize
-                continue
-            else:
+            if y_new != vnode.y or vnode.value:
```
Comment (vmarkovtsev) [inline lookout/style/format/code_generator.py]: We should not let NOOPS through because they are useless for code generation

Comment (zurk) [reply lookout/style/format/code_generator.py]: but it is just negation of the same condition it was before

Comment (vmarkovtsev) [reply lookout/style/format/code_generator.py]: Oh fuck yes

**Pull request merged**